src/olympia/devhub/fixtures/devhub/addon-validation-1.json

```diff
@@ -169,7 +169,7 @@
             "valid": true,
             "file": 100456,
             "notices": 1,
-            "validation": "{\"errors\": 1, \"detected_type\": \"extension\", \"success\": false, \"warnings\": 0, \"message_tree\": {\"testcases_targetapplication\": {\"__warnings\": 0, \"__errors\": 1, \"__messages\": [], \"__infos\": 0, \"test_targetedapplications\": {\"__warnings\": 0, \"__errors\": 1, \"__messages\": [], \"__infos\": 0, \"invalid_min_version\": {\"__warnings\": 0, \"__errors\": 1, \"__messages\": [\"d67edb08018411e09b13c42c0301fe38\"], \"__infos\": 0}}}}, \"infos\": 0, \"messages\": [{\"uid\": \"d67edb08018411e09b13c42c0301fe38\", \"tier\": 1, \"id\": [\"testcases_targetapplication\", \"test_targetedapplications\", \"invalid_min_version\"], \"file\": \"install.rdf\", \"message\": \"The value of <em:id> is invalid.\", \"context\": [\"<em:description>...\", \"<foo/>\"], \"type\": \"error\", \"line\": 0, \"description\": [\"<iframe>\", \"Version \\\"3.0b3\\\" isn't compatible with {ec8030f7-c20a-464f-9b0e-13a3a9e97384}.\"], \"signing_help\": [\"<script>&amp;\"]}], \"rejected\": false}\n"
+            "validation": "{\"errors\": 1, \"detected_type\": \"extension\", \"success\": false, \"warnings\": 0, \"message_tree\": {\"testcases_targetapplication\": {\"__warnings\": 0, \"__errors\": 1, \"__messages\": [], \"__infos\": 0, \"test_targetedapplications\": {\"__warnings\": 0, \"__errors\": 1, \"__messages\": [], \"__infos\": 0, \"invalid_min_version\": {\"__warnings\": 0, \"__errors\": 1, \"__messages\": [\"d67edb08018411e09b13c42c0301fe38\"], \"__infos\": 0}}}}, \"infos\": 0, \"messages\": [{\"uid\": \"d67edb08018411e09b13c42c0301fe38\", \"tier\": 1, \"id\": [\"testcases_targetapplication\", \"test_targetedapplications\", \"invalid_min_version\"], \"file\": \"install.rdf\", \"message\": \"The value of <em:id> is invalid. See <a href=\\\"https://mozilla.org\\\">mozilla.org</a> for more information\", \"context\": [\"<em:description>...\", \"<foo/>\"], \"type\": \"error\", \"line\": 0, \"description\": [\"<iframe>\", \"Version \\\"3.0b3\\\" isn't compatible with {ec8030f7-c20a-464f-9b0e-13a3a9e97384}.\"], \"signing_help\": [\"<script>&amp;\"]}], \"rejected\": false}\n"
```

Comment:
"...See <a href=\\\"https://mozilla.org\\\">mozilla.org</a> for more information\..." wut?

Reply:
Yeah, the escaping in this json file needs to be that weird because we're using double quotes that often.

Those are just test messages for the escaping tests in https://github.com/mozilla/addons-server/pull/12775/files#diff-5780aaf86f57e369e01dfa09883846a2R30-R193

Reply:
I couldn't see what tests used `https://mozilla.org` though - we don't reference that url in any of the devhub/submission tests.  As long as it's a valid result, fine.

Reply:
The tests linked above - https://github.com/mozilla/addons-server/pull/12775/files#diff-5780aaf86f57e369e01dfa09883846a2R30-R193 - do reference that URL.